server/graphql/core/src/generic_filters.rs

```diff
@@ -1,24 +1,26 @@
 use async_graphql::{InputObject, InputType};
 use chrono::{DateTime, NaiveDate, Utc};
-use repository::{DateFilter, DatetimeFilter, EqualFilter, SimpleStringFilter};
+use repository::{DateFilter, DatetimeFilter, EqualFilter, StringFilter};
 
 // simple string filter
 #[derive(InputObject, Clone)]
-pub struct SimpleStringFilterInput {
+pub struct StringFilterInput {
     /// Search term must be an exact match (case sensitive)
     equal_to: Option<String>,
-    /// Search term must be an exact match, but case insensitive
-    insensitive_equal_to: Option<String>,
```

Comment:
I couldn't find any uses for this in front end. @clemens-msupply could it be use in reports ?

Reply:
Just checked, there is no use of it. Please feel free to remove.